### ISSUE TITLE
The experimental result of the separation of two persons in svoice sisnr can reach "20.1". What are the audio numbers of tr, cv, and tt?

### ISSUE THREAD

**Issue by srdfjy** [facebookresearch/svoice#49]
Hi，

The experimental result of the separation of two persons in svoice sisnr can reach "20.1". What are the audio numbers of tr, cv, and tt?

Which signal-to-noise ratio range is the sample generated according to?

**Comment by adiyoss** [comment]
The SI-SNRi value that you mentioned (20.1) was reported over the test set (tt), a similar number was also reached for the validation set (cv), and I do not recall the train SI-SNRi numbers (probably a bit better).
Regrading SNR values, you can see all the details in our paper: http://proceedings.mlr.press/v119/nachmani20a/nachmani20a.pdf
Or you can find details on the dataset here: https://enk100.github.io/speaker_separation/

**Comment by srdfjy** [comment]
Hi，Are svoice's experimental results based on "make_dataset.py" to create a noisy data set for training?

**Comment by adiyoss** [comment]
@srdfjy,
In this paper: https://arxiv.org/pdf/2011.02329.pdf the noisy-reverberant data we generated using the `make_dataset.py` script. In this paper: http://proceedings.mlr.press/v119/nachmani20a/nachmani20a.pdf the dataset was generated using the scripts from this link https://enk100.github.io/speaker_separation/

**Comment by srdfjy** [comment]
thanks @adiyoss
